fluxy/fluxy.py

```diff
@@ -346,8 +346,22 @@ def to_flux(self) -> str:
         return f'sum(column: "{self.column}")'
 
 
+class Last:
+    def to_flux(self) -> str:
+        return "last()"
+
+
 Operation = (
-    AggregateWindow | Range | RangeOffset | Filter | Pivot | Drop | Keep | Map | Sum
+    AggregateWindow
+    | Range
+    | RangeOffset
+    | Filter
+    | Pivot
+    | Drop
+    | Keep
+    | Map
+    | Sum
+    | Last
 )
```

Comment:
We should probably define a `Protocol` for `to_flux`, then we wouldn't need this massive union

Reply:
Not sure what you mean here, do you mean define a `Protocol` for `Operation`, so that e.g. 

`Last(Operation)`
`AggregateWindow(PipedFunction, Operation)`

Reply:
See https://github.com/foundation-zero/fluxy/pull/4/commits/27599bf987eb6e2b1f35409b2a2d6ae0dd3adad5

Protocols can be implicit (meaning a structural check instead of a nominal check), so there's no need to include it in the inheritance list

Reply:
ah okay, because the main use is in a comprehension, so pyright just needs to know that self.operations contains a to_flux(), right?

`fluxes = (op.to_flux() for op in [self.initial, self.range, *self.operations])`

Reply:
Well, that every element in `self.operations` conforms to the `Operation` protocol by implementing a `to_flux()`, yes